prow/cluster/jobs/istio/envoy/istio.envoy.release-1.9.gen.yaml

```diff
@@ -29,7 +29,7 @@ periodics:
       - --modifier=merge_upstream_changes
       - --token-path=/etc/github-token/oauth
       - --merge-repository=https://github.com/envoyproxy/envoy.git
-      - --merge-branch="release/$(cut --fields=1,2 -d. VERSION)"
+      - --merge-branch="release/v1.17"
```

Comment:
This will introduce additional manual steps that will have to be done on branch cutting. The Envoy version doesn't match the Istio version, which we generally pull out and populate in https://github.com/istio/test-infra/blob/master/Makefile.core.mk for each branch. If we can't do that, I'd prefer having an automatic way to pull out the version like was present in the previous PR. 

I'm ok merging what is here right now as long as we can follow up and see if we can take care of this in some way that we don't expect someone to come in and make manual changes each release.